src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- You did well creating action types 👍, but according to the project requirements, the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like const ADD_BOOK = books/books/ADD_BOOK and const REMOVE_BOOK = books/books/REMOVE_BOOK. Please fix this 👍